src/element/QueryTable/model/QueryTable.ts

```diff
@@ -253,7 +253,8 @@ function sanitize_false_directives(sx_html: string): string {
 	});
 	const nodes_with_width = d_doc.querySelectorAll('*[width]');
 	nodes_with_width.forEach(node => node.removeAttribute('width'));
-	return d_doc.body.innerHTML;
+
+	return d_doc.body.childNodes[0];
```

Comment:
instead of assuming it only has 1 node, should just return d_doc.body.childNodes and have the calling function use ...

Reply:
updated based on recommendation 👍